lib/models/device.coffee

```
@@ -51,6 +51,10 @@ getDeviceModel = (deps, opts) ->
 
 	exports = {}
 
+	# Infer dashboardUrl from apiUrl if former is undefined
+	if not dashboardUrl?
+		dashboardUrl = apiUrl.replace(/api/, 'dashboard')
```

Should we just throw an exception here if `apiUrl` is not set?